docs/docs/concepts/persistence.md

```diff
@@ -259,16 +259,16 @@ memories[-1].dict()
  'updated_at': '2024-10-02T17:22:31.590605+00:00'}
 ```
 
-Each memory type is a Python class with certain attributes. We can access it as a dictionary by converting via `.dict` as above.
+Each memory type is a Python class ([`Item`](https://langchain-ai.github.io/langgraph/cloud/reference/sdk/python_sdk_ref/?h=item#langgraph_sdk.schema.Item)) with certain attributes. We can access it as a dictionary by converting via `.dict` as above.
```

Comment:
this should link to the regular `Item` here https://langchain-ai.github.io/langgraph/reference/store/#langgraph.store.base.Item, not the SDK Item